apps-rendering/src/components/NewsletterSignup/PrivacyWording.tsx

```diff
@@ -0,0 +1,76 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import { neutral, text, textSans } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { FC } from 'react';
+
+// ----- Component ----- //
+
+interface Props {
+	notUsingCaptcha: boolean;
+}
+
+const termsStyle = css`
+	${textSans.xxsmall({ lineHeight: 'tight' })}
+	color: ${text.supporting};
```

Comment:
```suggestion
	color: ${neutral[46]};
```

Reply:
sorry, I thought I'd changed that already!